### ISSUE TITLE
Travis fails due to timeout exceptions

### ISSUE THREAD

**Issue by tsiegleauq** [OpenSlides/OpenSlides#5046]
**Describe the bug**
Travis currently has a chance to fail due to timeout exceptions.
We can observe the problem for around two weeks by now. Restarting the build has a chance to get rid of the issue, but we should investigate why this happens in the first place.

According to my tests, the issue can only be observed inside of Travis. Local Karma-tests are running just fine.

**How to Reproduce**

1. Submit a PR

**Expected behavior**
The timeout exception should not occur

**Additional context**
```
Chrome 73.0.3683 (Linux 0.0.0) MotionsMotionSlideComponent should create FAILED
	Error: Timeout - Async callback was not invoked within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
	    at <Jasmine>
Chrome 73.0.3683 (Linux 0.0.0): Executed 284 of 383 (1 FAILED) (skipped 1) (0 secs / 3 mins 38.061 secs)
Chrome 73.0.3683 (Linux 0.0.0) MotionsMotionSlideComponent should create FAILED
	Error: Timeout - Async callback was not invoked within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
	    at <Jasmine>
```

**Screenshots**
![image](https://user-images.githubusercontent.com/10233032/65435722-42f2b480-de21-11e9-935a-d278f45ac13a.png)

**Comment by tsiegleauq** [comment]
Hopefully resolved by #5047 - keeping open to track further

**Comment by tsiegleauq** [comment]
Should be fixed now.
